### PR TITLE
Update code snippets

### PR DESCRIPTION
**Pull request by lukabratos**
----
CC @pusher/mobile

### CODE REVIEW

File: push-notifications-ios/push-notifications-ios/AppDelegate.swift [hunk 1]
```diff
@@ -7,18 +7,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     let pushNotifications = PushNotifications.shared
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        self.pushNotifications.start(instanceId: "f918950d-476d-4649-b38e-6cc8d30e0827")
+        self.pushNotifications.start(instanceId: "YOUR_INSTANCE_ID") // Can be found here: https://dash.pusher.com
```
Comment (luismfonseca) [inline push-notifications-ios/push-notifications-ios/AppDelegate.swift]: What's the motivation for this change?

Comment (lukabratos) [reply push-notifications-ios/push-notifications-ios/AppDelegate.swift]: To be consistent with the other SDKs and [docs](https://docs.pusher.com/beams/reference/ios#quickstart).
